pkg/storaged/storage-controls.jsx

```diff
@@ -82,31 +82,41 @@ function checked(callback) {
             return;
 
         const promise = client.run(callback);
-        if (promise)
+        if (promise) {
+            if (setSpinning)
+                setSpinning(true);
```

Comment:
Totally not important, but the kids these days use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) and write `setSpinning?.(true)`

Reply:
wow, that works with function calls?

Reply:
It's a kind of maaa-giiic :notes: